### ISSUE TITLE
TSLint error with JSONObject

### ISSUE THREAD

**Issue by gedennis** [Urigo/graphql-scalars#104]
Sorry to meet this issue, when I use JSONObject schema. Other scalars are okay.
```
 TS2322: Type '{ JSONObject: typeof GraphQLScalarType; DateTime: GraphQLScalarType; URL: GraphQLScalarType; Query: { ...}' is not assignable to type 'IResolvers<any, any> | IResolvers<any, any>[] | undefined'.
  Type '{ JSONObject: typeof GraphQLScalarType; DateTime: GraphQLScalarType; URL: GraphQLScalarType; Query: { ...}' is not assignable to type 'IResolvers<any, any>[]'.
    Property 'length' is missing in type '{ JSONObject: typeof GraphQLScalarType; DateTime: GraphQLScalarType; URL: GraphQLScalarType; Query: { ...'.
```
schema
```
import { DateTimeTypeDefinition, JSONObjectDefinition, URLTypeDefinition } from 'graphql-scalars';
export const SCHEMA = gql`
    ${SCALAR_DEFINITION}
    ${DateTimeTypeDefinition}
    ${URLTypeDefinition}
    ${JSONObjectDefinition}
   ...
```
Resolvers
```
import { DateTimeResolver, JSONObjectResolver, URLResolver } from 'graphql-scalars';

export const resolvers = {
    JSONObject: JSONObjectResolver,
};
```

**Comment by ardatan** [comment]
@gedennis Thank you! Could you try with the following canary version?
`1.0.2-61a23e8.0`

**Comment by gedennis** [comment]
@ardatan Definitely. I will try it with the new version. Thanks for your update.

**Comment by gedennis** [comment]
@ardatan It seems OK now. Could you like to release it to NPM ASAP?

**Comment by ardatan** [comment]
Available in 1.0.2!
